### ISSUE TITLE
Needed force feed all dinos

### ISSUE THREAD

**Issue by WeWyaha** [VoidArkMods/QoLPlus#157]
I been breeding dinos with love potion in they're inventory but it is tiring going one by one to force feed each dino to mate again with it.is it possible to add a remote to force feed all dinos? So I don't have to go one by one to feed love potions and mutator potions

**Comment by penash42** [comment]
This mod doesn't add potions.  Probably better to direct this suggestion to whatever potion mod you're using.

**Comment by WeWyaha** [comment]
> This mod doesn't add potions. Probably better to direct this suggestion to whatever potion mod you're using.

Yes I know that.im saying is there a way to force feed dinos all at once.i know it doesn't have potions but I'm using potions love potion to feed dinos one by one for them to enable mating again

**Comment by VoidArkMods** [comment]
Hello!
It sounds like a feature-request, not really an issue. But I get what you mean: some sort of "remote force feed item" on the multi-tool or something similar. I'll consider it, but it is going to be quite low on the priority list unfortunately.
